docs/docs/recipes.md

```diff
@@ -32,56 +32,96 @@ Craving a happy medium between [full-length tutorials](/tutorial/) and crawling
 
 ## Table of Contents
 
-- [Using Gatsby without Graphql](#using-gatsby-without-graphql)
-- [Gatsby project structure](#gatsby-project-structure)
-- [Using a starter](#using-a-starter)
-- [Creating pages](#creating-pages)
-- [Linking between pages](#linking-between-pages)
-- [Styling](#styling)
-- [Creating layouts](#creating-layouts)
-- [Deploying](#deploying)
-- [Querying data](#querying-data)
-- [Sourcing data](#sourcing-data)
-- [Transforming data](#transforming-data)
+1. [Pages and Layouts](#pages-and-layouts)
+2. [Styling with CSS](#styling-with-css)
+3. [Working with starters](#working-with-starters)
+4. [Working with themes](#working-with-themes)
+5. [Sourcing data](#sourcing-data)
+6. [Querying data](#querying-data)
+7. [Working with images](#working-with-images)
+8. [Transforming data](#transforming-data)
+9. [Deploying your site](#deploying-your-site)
 
-## Using Gatsby without GraphQL
+## Pages and Layouts
```

Comment:
This is a huge section. It would be cool to have sub-items in the TOC, but this can probably wait for the TOC component.